packages/npm/@amazeelabs/gatsby-source-silverback/package.json

```diff
@@ -6,29 +6,30 @@
   "author": "Amazee Labs <development@amazeelabs.com>",
   "license": "MIT",
   "private": false,
+  "type": "module",
   "publishConfig": {
     "access": "public"
   },
   "devDependencies": {
     "@amazeelabs/eslint-config": "1.4.43",
     "@amazeelabs/prettier-config": "1.1.3",
     "@rollup/plugin-node-resolve": "15.1.0",
+    "@types/isomorphic-fetch": "0.0.36",
     "@types/react": "18.2.12",
     "eslint": "8.42.0",
     "gatsby": "5.10.0",
     "prettier": "2.8.8",
     "react": "18.2.0",
-    "rollup": "3.25.1",
-    "rollup-plugin-esbuild": "5.0.0",
     "typescript": "5.1.3"
   },
   "scripts": {
-    "build": "rollup -c rollup.config.ts --configPlugin typescript",
```

Comment:
Rollup bundles everything into one file, which is not necessary with the switch to ESM. And `tsc` was the simplest possible solution. Since code is not going to grow a lot and this transpile is not run all the time, I felt adding another tool was not necessary. (I am aware that I used `swc` in such cases before, but I am allowed to learn from my mistakes 😛 )